modules/sbo-data-model.adoc

```diff
@@ -1,9 +1,14 @@
 // Module included in the following assemblies:
 //
-// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+// * /applications/connecting_applications_to_services/exposing-binding-data-from-a-service.adoc
+
+:_content-type: CONCEPT
 [id="sbo-data-model_{context}"]
 = Data model
 
+[role="_abstract"]
+This section explains the data model used in the annotations and OLM descriptors.
```

Comment:
Generally, RH docs avoid self-reference phrases like "This section explains..."
Does the following suggestion accurately reflect the purpose of this topic?
```suggestion
The data model used in the annotations and OLM descriptors follow specific conventions.
```

Reply:
IBM SG: Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes".
Instead, focus on the purpose or value of the content, and include words that users are likely to search
for. For example, in a short description in an online help topic, provide users with enough information to
determine whether the topic meets their needs. You can use self-referential text in learning modules
and tutorials.